lib/internal/http2/core.js

```diff
@@ -2214,6 +2176,30 @@ class Http2Stream extends Duplex {
   }
 }
 
+function onTimeout(kSession) {
```

Comment:
nit, it would arguably be nicer to accept the context in rather than use `.call` in both instances.

Reply:
fixed